### ISSUE TITLE
Web UI Lags Behind

### ISSUE THREAD

**Issue by SurenFTN** [blockscout/blockscout#10045]
### Description

https://www.ftnscan.com/  The Web ui is showing the transaction with lag approx two minutes below Is the .env realted to Indexer
[1.txt](https://github.com/blockscout/blockscout/files/15271555/1.txt)
I have also added some config in Posgtres  
 shared_buffers = '64GB';
 temp_buffers = '8MB'; 
 work_mem = '1GB';
 maintenance_work_mem = '32GB';
 effective_cache_size = '32GB';

PS My Server's Specs are 
64x core cpu
128gb ram 

### Type of the installation

Manual from the source code

### Type of the JSON RPC archive node

Geth

### Type of the chain

L1

### Link to the page

https://www.ftnscan.com/

### Backend version

V5.2.2-beta

### Elixir & Erlang/OTP versions

elixir 1.13.4-otp-24/erlang 24.0

### Operating system

Linux

**Comment by vbaranov** [comment]
@SurenFTN 
> V5.2.2-beta

is too old. Please migrate to the [latest](https://github.com/blockscout/blockscout/releases/tag/v6.5.0-beta) release. It addresses different performance aspects of the on-chain data import.

**Comment by SurenFTN** [comment]
Thank you @vbaranov . Can you leave this issue open for couple of days unitl I update the backend. Thank you in advance

**Comment by BlockgenStudio** [comment]
Hi @vbaranov we also run into similar problem. it seems take a really long time to load transaction: e.g. http://supernettestnet.explorer.arianee.com/tx/0xabf152bcc152968323f692716ee89cf763c4f773dff611bd07dad4d15c787452
any suggestion to improve the performance?

Here is our set up:

Type of the installation
Manual from the source code

Type of the JSON RPC archive node
Geth

Type of the chain
L2

Link to the page
No response

Steps to reproduce
No response

Backend version
v6.2

Frontend version
No response

Elixir & Erlang/OTP versions
1.14.5

Operating system
linux

Env.
INDEXER_MEMORY_LIMIT = 15

**Comment by SurenFTN** [comment]
@vbaranov I have updated to 6.5.0 but encoutner antoher issue the Indexing of internal transactions is too low  in the block's table's number count is approx 150k but actual number i more than 2.5M so without fully sincing I cannot check wether the whole indexer is lagging or not

**Comment by SurenFTN** [comment]
I have a porblem in syning I have already set the value yout told ETHEREUM_JSONRPC_GETH_TRACE_BY_BLOCK=true but in the blocks table still the count is behind for approx 400k block an its not indexing I have attached the logs
[Uploading prod.zip…]()

**Comment by vbaranov** [comment]
@SurenFTN was the issue resolved?

**Comment by SurenFTN** [comment]
yep you can close it "debug_traceBlockBynumber" with this method  its syncing ideally thank you @vbaranov
